### PR TITLE
aquaskk: new recipe

### PR DESCRIPTION
**Pull request by hanya**

### CODE REVIEW

**Comment by waddlesplash** [comment]
Looks good to me.

So there will now be 3 Japanese input methods in HaikuPorts: Canna, Mozc, and this one. I think we currently ship Canna with release builds of Haiku (or maybe not? looks like we forgot to do that for R1/beta1). But I guess that one is pretty old and hasn't seen much attention lately.

Do you have any suggestions for which one we should bundle with Haiku by default?

File: app-i18n/aquaskk/aquaskk-4.7.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	$portPackageLinksDir/cmd~python2/bin/python2 build.py gyp \
+		--platform=Haiku --arch=$targetArchitecture
+
+	$portPackageLinksDir/cmd~python2/bin/python2 build.py build \
+		-c Release \
+		method/method.gyp:aquaskk \
+		preference/preference.gyp:preference
+}
+
+INSTALL()
```
Comment (extrowerk) [inline app-i18n/aquaskk/aquaskk-4.7.0.0.recipe]: Maybe you should provide a makefile as additional file instead this.

File: app-i18n/aquaskk/aquaskk-4.7.0.0.recipe [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+	cp $AOutDir/aquaskk $InputMethodDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	cp $AOutDir/aquaskk_task $DataDir
+fi
+	cp $AOutDir/aquaskk_preference $DataDir
+
+	cp $APlatDir/src/preference/dict_downloader.py $DataDir
+
+	cp $AConfigDir/act.conf $DataDir
```
Comment (extrowerk) [inline app-i18n/aquaskk/aquaskk-4.7.0.0.recipe]: You can write

cp $AConfigDir/{act.conf,act.rule} $DataDir

So you can copy more than one file from the same source folder to the same target folder.

File: app-i18n/aquaskk/aquaskk-4.7.0.0.recipe (hunk 1 above)
Comment (waddlesplash) [reply app-i18n/aquaskk/aquaskk-4.7.0.0.recipe]: Huh? Gyp works fine, there's no need for that.

**Comment by hanya** [comment]
Last fix contains crash while uninstall package.

**Comment by hanya** [comment]
> Do you have any suggestions for which one we should bundle with Haiku by default?

SKK way to input is not normal. AquaSKK should not be default one.

**Pull request merged**